src/validate.ts

```diff
@@ -65,16 +65,8 @@ export function validate(toolOutput: LSIF.Element[], ids: string[]): number {
     checkAllVisited();
 
     if (fse.pathExistsSync(protocolPath)) {
-        checkVertices(ids.filter((id: string) => {
-            const element: LSIF.Element = toolOutput.filter((e: LSIF.Element) => e.id.toString() === id)[0];
-
-            return element.type === 'vertex';
-        }));
-        checkEdges(ids.filter((id: string) => {
-            const element: LSIF.Element = toolOutput.filter((e: LSIF.Element) => e.id.toString() === id)[0];
-
-            return element.type === 'edge';
-        }));
+        checkVertices(toolOutput.filter((e : LSIF.Element) => e.type === 'vertex').map(e => e.id.toString()));
```

Comment:
Quick suggestion to adhere to TSLint:

```suggestion
        checkVertices(toolOutput.filter((e : LSIF.Element) => e.type === 'vertex')
        .map((e: LSIF.Element) => e.id.toString()));
```